src/services/storage.service.ts

```diff
@@ -13,4 +13,8 @@ export const StorageService = {
   async getStorageLocations(): Promise<IStorageLocation[]> {
     return (await sdk.database.listDocuments<IStorageLocation>(STORAGE_LOCATION_COLLECTION_ID)).documents;
   },
+  async getStorageLocationById($id: string): Promise<IStorageLocation> {
+    const storageLocation = await sdk.database.getDocument<IStorageLocation>(STORAGE_LOCATION_COLLECTION_ID, $id);
+    return storageLocation;
```

Comment:
Don't use a variable here. Return it directly.
```suggestion
    return await sdk.database.getDocument<IStorageLocation>(STORAGE_LOCATION_COLLECTION_ID, $id);
```